distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/config/ConfigureTestCommons.java

```diff
@@ -13,71 +13,110 @@
  */
 package org.codice.ddf.itests.common.config;
 
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static org.awaitility.Awaitility.with;
+
 import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collections;
 import java.util.Dictionary;
 import java.util.List;
+import java.util.concurrent.TimeUnit;
 import org.codice.ddf.configuration.DictionaryMap;
 import org.codice.ddf.itests.common.AdminConfig;
 import org.osgi.service.cm.Configuration;
 
 public class ConfigureTestCommons {
 
-  public static final String METACARD_VALIDITITY_FILTER_PLUGIN_SERVICE_PID =
+  private static final String METACARD_VALIDITY_FILTER_PLUGIN_SERVICE_PID =
       "ddf.catalog.metacard.validation.MetacardValidityFilterPlugin";
 
-  public static final String METACARD_VALIDITITY_MARKER_PLUGIN_SERVICE_PID =
+  private static final String METACARD_VALIDITY_MARKER_PLUGIN_SERVICE_PID =
       "ddf.catalog.metacard.validation.MetacardValidityMarkerPlugin";
 
-  public static final String METACARD_ATTRIBUTE_SECURITY_POLICY_PLUGIN_PID =
+  private static final String METACARD_ATTRIBUTE_SECURITY_POLICY_PLUGIN_PID =
       "org.codice.ddf.catalog.security.policy.metacard.MetacardAttributeSecurityPolicyPlugin";
 
-  public static final String AUTH_Z_REALM_PID = "ddf.security.pdp.realm.AuthzRealm";
+  private static final String AUTH_Z_REALM_PID = "ddf.security.pdp.realm.AuthzRealm";
 
   private ConfigureTestCommons() {}
 
-  public static void configureMetacardValidityFilterPlugin(
-      List<String> securityAttributeMappings, AdminConfig configAdmin) throws IOException {
-    Configuration config =
-        configAdmin.getConfiguration(METACARD_VALIDITITY_FILTER_PLUGIN_SERVICE_PID, null);
-    Dictionary<String, Object> properties = new DictionaryMap<>();
-    properties.put("attributeMap", securityAttributeMappings);
-    config.update(properties);
+  public static Dictionary<String, Object> configureService(
```

Comment:
📔 To me, this is a case where defining this functionality as a JUnit rule and letting the rule automatically restore configurations properly at the end of a test wether it failed or not is more practical and safe then relying on each tests to restore properly config in all possible cases